game/graphic/Size.cc

```diff
@@ -0,0 +1,24 @@
+#include "Size.hh"
```

Comment:
Personally i'm fine with PascalCasing filenames and classes/interfaces and I actually prefer it, but we haven't done that before. We used snake_casing up until now. Not sure what others think of that @Lord-Kamina @ooshlablu

Reply:
That is a good point. I like if file name is exact the class name. Actual some classes are "hidden" behind file names, but may be file names follow no scheme ... (f.e. the screen classes)

Reply:
Personally i'm a fan of having one file per class/header/test

That might be another refactor as you say some classes are hidden which isn't that nice

Reply:
The current naming is not consistent. Always lower case, sometimes with _, but not always snake. 

My opinion: I make filenames lowercase in this pr and would love if we can go the way @Lord-Kamina  suggested: extra pr switching to PascalCase :+1: :+1:

Reply:
:+1: @twollgam

Reply:
done